### PR TITLE
📝 add toast documentation

### PR DESCRIPTION
**Pull request by YounesBessa**

### CODE REVIEW

**Review by jerembdn — APPROVED** [review]

File: docs/pages/docs/components/toast.mdx [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+
+import Playground from "@components/Playground";
+
+# Toast
+
+A concise, temporary message.
+
+## Usage
+
+First of all, you need to import the `Toast` component from the `@tonightpass/kitchen` package.
```
Comment (AntoineKM) [inline docs/pages/docs/components/toast.mdx:14]: importing toast comp isn't need but the useToats hook yep

File: docs/pages/docs/components/toast.mdx [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+}
+`}
+/>
+
+## Cancel
+
+<Playground
+  code={`
+  () => {
+  const { setToast } = useToasts();
```
Comment (AntoineKM) [inline docs/pages/docs/components/toast.mdx:119]: add spaces between each part of code

File: docs/pages/docs/components/toast.mdx [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+`}
+/>
+
+## Action
+
+<Playground
+  code={`
+  () => {
+  const { setToast } = useToasts();
+  const alertAction = {
```
Comment (AntoineKM) [inline docs/pages/docs/components/toast.mdx:94]: add spaces between each part of code

File: docs/pages/docs/components/toast.mdx [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+}
+`}
+/>
+
+## Types
+
+<Playground 
+  code={`
+  () => {
+  const { setToast } = useToasts();
```
Comment (AntoineKM) [inline docs/pages/docs/components/toast.mdx:150]: add spaces between each part of code

**Review by AntoineKM — CHANGES_REQUESTED** [review]

**Review by AntoineKM — APPROVED** [review]

**Pull request merged**